apps/browser/src/autofill/services/autofill.service.ts

```diff
@@ -1343,10 +1358,17 @@ export default class AutofillService implements AutofillServiceInterface {
    * @returns {boolean}
    * @private
    */
-  private isExcludedType(type: string, excludedTypes: string[]) {
+  static isExcludedType(type: string, excludedTypes: string[]) {
     return excludedTypes.indexOf(type) > -1;
   }
 
+  static isSearchField(field: AutofillField) {
+    const matchFieldAttributeValues = [field.type, field.htmlName, field.htmlID, field.placeholder];
+    const matchPattern = new RegExp(AutoFillConstants.SearchFieldNames.join("|"), "gi");
+
+    return !!matchFieldAttributeValues.join(" ").match(matchPattern);
```

Comment:
```suggestion
    return Boolean(matchFieldAttributeValues.join(" ").match(matchPattern));
```

Nit, no changes required. Any thoughts on using a `Boolean` class constructor instead of a double bang `!!` for type conversion to boolean?

Reply:
1cc7c1cb1b50fb3c89f5229ddc457b399e658f6f 👍